sdk/resourcemanager/Azure.ResourceManager.Core/src/ContainerBase.cs

```diff
@@ -26,6 +26,7 @@ protected ContainerBase()
         internal ContainerBase(ClientContext clientContext)
             : base(clientContext, ResourceIdentifier.RootResourceIdentifier)
         {
+            Parent = new TenantOperations(clientContext);
```

Comment:
not sure this is safe in all cases

Reply:
Reverted this change in commit ab2937234b314fca46e31b20b27b2ae670bb1b05. I think the right fix to the original problem is to use `new PolicyAssignment(this, response.Value)` instead of `new PolicyAssignment(Parent, response.Value)` in `PolicyAssignmentContainer`. I also removed the PolicyAssignmentContainer constructor with the parent parameter.